src/Analysis/Ast/Test/LintTypeVarTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Python.Analysis.Diagnostics;
+using Microsoft.Python.Analysis.Tests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TestUtilities;
+
+namespace Microsoft.Python.Analysis.Tests {
+
+    [TestClass]
+    public class LintTypeVarTests : AnalysisTestBase {
+
+        internal static class Utils {
```

Comment:
I would delete this whole thing, but a couple style comments:

You can just put this at the top level, you don't need some hidden static class for this.

C# constants don't use the ALL_CAPS_NAME format, that's a Java thing. You'd probably do `TypeVarImport` and leave it at that.

Reply:
Ok thanks for the tips!